src/components/Card/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, ReactNode } from 'react';
+import classes from './Card.module.scss';
+
+ type CardProps = {
+  title: string;
+  data: {
+    [k: string]: string;
+  }[];
+ }
+
+const Card: FC<CardProps> = ({title, data}) => {
+      const categoryItems = data.map((item) => 
```

Comment:
move this map inside the JSX code. it is not mandatory of course, but having this map call on it's place in the JSX increases readability a little and looks better when JSX is in the JSX))

Reply:
also I believe the name of the variable is better to be changed))

Reply:
Hope this changes is ok